packages/manager-react-components/src/components/content/dashboard-tile/tile-block.component.tsx

```diff
@@ -11,9 +11,11 @@ export const TileBlock: React.FC<React.PropsWithChildren<TileBlockProps>> = ({
   children,
 }) => (
   <dl className="flex flex-col gap-1 m-0">
-    <dt>
-      <OdsText preset={ODS_TEXT_PRESET.heading6}>{label}</OdsText>
-    </dt>
+    {label && (
```

Comment:
```suggestion
    {!!label && (
```
If label is a string it will display both

Reply:
I agree it's better that the boolean check is explicit, however I'm pretty sure it won't have this behavior you describe. Anyway I updated it !